ui/compontents.go

```diff
@@ -676,6 +676,20 @@ func (page *pageCommon) walletInfoPopup(gtx layout.Context) layout.Dimensions {
 	})
 }
 
+func (page pageCommon) addAccount(account walletAccount) {
+	account.evt = &gesture.Click{}
+
+	if len(page.wallAcctSelector.walletAccounts.selectSendAccount) > account.walletIndex {
+		account.accountIndex = len(page.wallAcctSelector.walletAccounts.selectSendAccount[account.walletIndex])
```

Comment:
`page.wallAcctSelector.selectedSendWallet` and `page.wallAcctSelector.selectedSendAccount` can be assigned to variables to make the lines shorter horizontally and easier to read.